### ISSUE TITLE
docker build of eosio/builder image fails

### ISSUE THREAD

**Issue by betafive** [EOSIO/eos#4552]
We prefer to build our own docker images from scratch rather than pulling unverified content from the Docker Hub. So we first build our own `eosio/builder` image and then use this to build an `eosio/eos` image.

Attempting to build `eosio/builder` using v1.0.8 tag currently fails:

```
git clone https://github.com/eosio/eos.git -b v1.0.8 --recursive
cd eos/Docker/builder/
docker build .

<snip>

Step 14/16 : RUN git clone --depth 1 -b releases/stable https://github.com/mongodb/mongo-cxx-driver     && cd mongo-cxx-driver     && cmake -H. -Bbuild -G Ninja -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release  -DCMAKE_INSTALL_PREFIX=/usr/local    && cmake --build build --target install     && cd .. && rm -rf mongo-cxx-driver
 ---> Running in 9777f8a57e4e
Cloning into 'mongo-cxx-driver'...
-- The CXX compiler identification is Clang 4.0.1
-- Check for working CXX compiler: /usr/bin/clang++
-- Check for working CXX compiler: /usr/bin/clang++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- The C compiler identification is Clang 4.0.1
-- Check for working C compiler: /usr/bin/clang
-- Check for working C compiler: /usr/bin/clang -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Auto-configuring bsoncxx to use MNMLSTC for polyfills since C++17 is inactive
CMake Error at src/bsoncxx/CMakeLists.txt:86 (find_package):
  Could not find a configuration file for package "libbson-static-1.0" that
  is compatible with requested version "1.10.0".

  The following configuration files were considered but not accepted:

    /usr/local/lib/cmake/libbson-static-1.0/libbson-static-1.0-config.cmake, version: 1.9.3



-- Configuring incomplete, errors occurred!
See also "/mongo-cxx-driver/build/CMakeFiles/CMakeOutput.log".
The command '/bin/sh -c git clone --depth 1 -b releases/stable https://github.com/mongodb/mongo-cxx-driver     && cd mongo-cxx-driver     && cmake -H. -Bbuild -G Ninja -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release  -DCMAKE_INSTALL_PREFIX=/usr/local    && cmake --build build --target install     && cd .. && rm -rf mongo-cxx-driver' returned a non-zero code: 1
```

**Comment by betafive** [comment]
@dhaneshvb This is a bug not a question - trying to build the docker image fails.

**Comment by jgiszczak** [comment]
Try building v1.1.0 instead.  There have been some Docker issues fixed in releases subsequent to 1.0.8.

**Comment by betafive** [comment]
This worked today but there's no guarantee it will continue to do so as the build scripts do not specify exact versions of dependencies (see https://github.com/EOSIO/eos/issues/4735). Closing this issue in favour of 4735...
